legate/tester/stages/test_stage.py

```diff
@@ -205,6 +205,24 @@ def file_args(self, test_file: Path, config: Config) -> ArgList:
 
         return args
 
+    def cov_args(self, config: Config) -> ArgList:
+        """Coverage binary and coverage arguments.
+
+        Parameters
+        ----------
+        config: Config
+            Test runner configuration
+
+        """
+        if config.cov_bin:
+            args = [str(config.cov_bin)] + config.cov_args.split()
+            if config.cov_src_path:
+                args += ["--source=%s" % config.cov_src_path]
```

Comment:
prefer f-strings in new code
```suggestion
                args += [f"--source={config.cov_src_path}"]
```